gravitee-am-certificate/gravitee-am-certificate-api/src/main/java/io/gravitee/am/certificate/api/CertificateProvider.java

```diff
@@ -45,6 +46,10 @@ public interface CertificateProvider {
 
     String signatureAlgorithm();
 
+    default Maybe<CertificateBundle> getCertificateBundleDetails() {
+        return Maybe.never();
```

Comment:
**question**: should we return Maybe.empty instread of never ? From my understanding never will never finish.

Reply:
you're right.